src/actions/loadData.js

```diff
@@ -61,9 +92,13 @@ const fetchDataAndDispatch = (dispatch, url, query, narrativeBlocks) => {
     fetchExtras += `&deprecatedSecondTree=${query.tt}`;
   }
 
-
-  // fetchJSON(`${charonAPIAddress}request=mainJSON&url=${url}${fetchExtras}`)
   getDataset(`${url}${fetchExtras}`)
+    .then((response) => {
+      url = response.url.split('prefix=')[1];
```

Comment:
Currently `prefix=X` is the last part of the query, so this technique works, but if there were to be another part to the query this wouldn't work as desired. Better to use `queryString` to parse the query I think.